packages/notifications/src/utils/icons.ts

```diff
@@ -11,7 +11,7 @@ import AlertWarningStrokeIcon from '@zendeskgarden/svg-icons/src/16/alert-warnin
 import InfoStrokeIcon from '@zendeskgarden/svg-icons/src/16/info-stroke.svg';
 import { VALIDATION_TYPE } from './types';
 
-export const validationIcons: Record<VALIDATION_TYPE, object> = {
+export const validationIcons: Record<VALIDATION_TYPE, Record<string, unknown>> = {
```

Comment:
Updates have been made to the types using `{}` and `object`. The defaults for `@typescript-eslint/ban-types` have been [updated](https://github.com/typescript-eslint/typescript-eslint/pull/848) to warn when using these types, and provides suggestions:

<img width="897" alt="Screen Shot 2020-06-04 at 3 32 19 PM" src="https://user-images.githubusercontent.com/1811365/83818776-06987900-a67d-11ea-9fe4-b0ddf0cb887a.png">